tests/lora/test_lora_layers_peft.py

```diff
@@ -1106,7 +1178,7 @@ def test_integration_logits_multi_adapter(self):
             output_type="np",
         ).images
         predicted_slice = images[0, -3:, -3:, -1].flatten()
-        expected_slice_scale = np.array([0.5977, 0.5985, 0.6039, 0.5976, 0.6025, 0.6036, 0.5946, 0.5979, 0.5998])
+        expected_slice_scale = np.array([0.5888, 0.5897, 0.5946, 0.5888, 0.5935, 0.5946, 0.5857, 0.5891, 0.5909])
```

Comment:
@younesbelkada why are these values being changed?

Reply:
because the previous API was broken and the values were wrong :/ so this PR updates the slow tests with the correct values